india_compliance/gst_india/doctype/gstr_3b_report/test_gstr_3b_report.py

```
@@ -758,6 +767,126 @@ def test_rcm_outward_liability(self):
+
+        The commit itself is skipped under frappe.flags.in_test, so this asserts
+        the write rather than its durability.
+        """
+        with patch("frappe.enqueue_doc"):
+            report = self.create_report(enqueue_report=1).insert()
+
+        with (
+            patch.object(GSTR3BReport, "_process_outward_itc", side_effect=Exception("boom")),
+            patch("frappe.publish_realtime") as publish_realtime,
+        ):
+            self.assertRaises(Exception, report.get_data)
```

Ruff (0.15.21)

[warning] Do not assert blind exception: `Exception`

(B017)